.circleci/config.yml

```diff
@@ -162,25 +163,25 @@ jobs:
           echo $PGP_SECRET | base64 --decode | gpg --import --no-tty --batch --yes
       - run: sbt clean ci-release
 
-  microsite:
-    docker:
-      - image: cimg/node:16.13
-    steps:
-      - checkout
-      - run: |
-          if [ "$(git diff --numstat HEAD^ -- vuepress)" ]; then
-            cd vuepress
-            npm install
-            npm run docs:build
-            cd ..
-            rm -rf docs
-            cp -r vuepress/docs/.vuepress/dist docs
-            git add docs
-            git config user.name "Pierre Ricadat"
-            git config user.email "ghostdogpr@gmail.com"
-            git commit -m "[ci skip] update docs"
-            git push
-          fi
+#  microsite:
```

Comment:
I am disabling the docs publishing in the CI in order to merge this PR, I will re-enable it when we release.
That way we can update docs for now and it won't be reflected on the site yet.